scripts/install.sh

```
@@ -41,23 +57,31 @@ die()   { printf "  ${C_ERR}✗  %s${C_RST}\n" "$1" >&2; exit 1; }
+    fi
+    if [ "$_expect" = "version" ]; then
+        _VERSION_ARG="$arg"; _expect=""
         continue
     fi
     case "$arg" in
-        --verbose|-v) _VERBOSE=true ;;
-        --python)     _next_is_python=true ;;
+        --binary)        MODE="binary" ;;
+        --source)        MODE="source" ;;
+        --version)       _expect="version" ;;
+        --verbose|-v)    VERBOSE=true ;;
```

Shellcheck (0.11.0)

[warning] VERBOSE appears unused. Verify use (or export if used externally).

(SC2034)